### ISSUE TITLE
The image zoom controls are unlabeled 

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#24494]
**Describe the bug**
IN WordPress 5.5 the block editor provides new UIs to edit images in the post context. Amongst them, there's a UI to "zoom" the image. In this mini-UI both the range and the number input are unlabeled. No label element, no aria-label or other labelling. Thus, there's no accessible name assistive technologies can use to announce _what_ these controls are.

Range input:

<img width="1282" alt="range" src="https://user-images.githubusercontent.com/1682452/89925199-5ce8e180-dc03-11ea-99c3-775b34e05061.png">

Number input:

<img width="1276" alt="number" src="https://user-images.githubusercontent.com/1682452/89925221-65411c80-dc03-11ea-9d8c-10e48a00a40c.png">

All interactive elements must have an accessible name. Form controls must be labeled.

For accessibility, these are the recommended ways to label form elements in order of preference:
- a visible `<label>` element
- a visually hidden `<label>` element
- an `aria-label` attribute
- an `aria-labelledby` attribute but only if it points to some meaningful text already present in the page

Aside: I thought there used to be checks to prevent unlabeled form controls, but maybe I'm missing something.

Additionally;
The markup is invalid, as:
- There are `<div>` elements wrapped in a `<span>` element.
- A `color` prop is rendered in the HTML.
- The `inputmode` attribute isn't standard yet: it is in [the HTML Living Standard spec](https://html.spec.whatwg.org/multipage/interaction.html#input-modalities:-the-inputmode-attribute) but it's not present in [the current W3C HTML 5.2 recommendation](https://www.w3.org/TR/html52/editing.html#user-interaction-editing). Worth also noting `inputmode` is not supported by all browsers, most notably not supported by Safari, IE11, Firefox for Android. See https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode

Passing the related rendered HTML to the W3C validator:

<img width="1416" alt="zoom ui invalid markup" src="https://user-images.githubusercontent.com/1682452/89927590-d33b1300-dc06-11ea-85a3-2a05978ace2d.png">

**Comment by afercia** [comment]
Found the reference to the automated tests to make sure all fields include labels: https://github.com/WordPress/gutenberg/issues/6098

Not sure id anything has changed or if it applies to this specific case but it would be nice to further iterate and make sure there are tools in place to avoid unlabeled controls.
